porch/func/server/server.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"net"
+
+	pb "github.com/GoogleContainerTools/kpt/porch/func/evaluator"
+	"google.golang.org/grpc"
+)
+
+func main() {
+	lis, _ := net.Listen("tcp", fmt.Sprint(":9445"))
+	server := grpc.NewServer()
+	pb.RegisterFunctionEvaluatorServer(server, &evaluator{})
+	server.Serve(lis)
```

Comment:
We probably should check/log errors, but we can do that later...

Reply:
ack, will do.